<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <link rel="stylesheet" href="../../css/bootstrap.min.css">
    <link rel="stylesheet" href="../../css/bootstrap-grid.min.css">
    <link rel="stylesheet" href="../../css/template.css">

    <script src="../../js/jquery.min.js"></script>
    <script src="../../js/bootstrap.min.js"></script>
    <style>
      .TITRE, .intro, {text-align: center;}
      .overview, {background-image:url("./Foliar_overview.jpg"); margin-left:auto; margin-right: auto;}

    </style>
    <title>Protocols</title>
  </head>
  <body>
    <h1 class="TITRE">Protocols</h1>
    <h2 class="TITRE">Protocol 1 : culture medium preparation</h2>
    <p><strong>Aim :</strong> Grow bacteria and/or select a recombinant strain</p>
    <h3 class="TITRE"><strong>1. Liquid LB Medium (500ml)</strong></h3>
    <p>Mix the following component in 500mL of water :</p>
    <ul>
      <li>5g tryptone</li>
      <li>2,5g yeast extract</li>
      <li>5g NaCl</li>
    </ul>
    <p>Add a magnetic stirrer bar and mix. Name the bottle and close it with foil paper (you can also use a bottle cap). Be careful to not fill the bottle entirely otherwise it might overflow during the autoclave step !</p>
    <p>Autoclave at 121°C for 30 minutes. Allow the autoclaved medium to cool to 55ºC and add the antibiotic if necessary :</p>
    <ul>
      <li>Chloramphenicol 35µg/ml</li>
      <li>Ampicillin at 100µg/ml</li>
      <li>Kanamycin 35µg/ml</li>
    </ul>
    <p>Store under the BSC or at 4°C.</p>
    <p><strong>Source: </strong>
    <a href="http://www.sigmaaldrich.com/technical-documents/articles/biology/microbial-media.html">"http://www.sigmaaldrich.com/technical-documents/articles/biology/microbial-media.html"</a></p>
    <p><strong>Reagents:</strong></p>
    <ul>
      <li>Tryptone (VWR 84610.0500)</li>
      <li>Yeast extract (Sigma-Aldrich Y1625-250G)</li>
      <li>NaCl Labkem (SOCH-00T-1K0)</li>
      <li>Distilled water </li>
      <li>Chloramphenicol (Sigma-Aldrich C0378-25G)</li>
      <li>Ampicillin (Sigma-Aldrich A9393-5G)</li>
      <li>Kanamycin (Sigma-Aldrich 60615-5G)</li>
    </ul>
    <p><strong>Materials: </strong></p>
    <ul>
      <li>Autoclave (Nanbei LS-100LJ)</li>
      <li>Steam indicator stips</li>
      <li>Magnetic stirrer Bioblock scientific 94361</li>
      <li>Water purification system Merck Millipore Elix 3</li>
      <li>Analytical balance Ohaus Discovery</li>
      <li>Biosafety cabinet ESI-Flufrance Cytair 125</li>
    </ul>
    <h3 class="TITRE"><strong>2. LB Agar plates preparation (500ml)</strong></h3>
    <p>Mix the following component in 500mL of water :</p>
    <ul>
      <li>5g tryptone</li>
      <li>2,5g yeast extract</li>
      <li>5g NACL</li>
      <li>7g of agar powder</li>
    </ul>
    <p>Add a magnetic stirrer bar and mix. Name the bottles and close them with foil paper (you can also use a bottle cap). Be careful to not fill the bottle entirely otherwise it might overflow !
       Autoclave at 121°C for 30 minutes. Allow the autoclaved medium to cool to 55ºC (be careful not to wait too long because the gel will polymerise !) and add the antibiotic if necessary :
    </p>
    <ul>
      <li>Chloramphenicol 35µg/ml</li>
      <li>Ampicillin at 100µg/ml</li>
      <li>Kanamycin 35µg/ml</li>
    </ul>
    <p>Let the plates dry under the BSC (Biosafety Cabinet) then store the plates at 4°C for later uses. If you plan to use the dishes you can also let them into the BSC but they will quickly overdry.</p>
    <p><strong>Source: </strong><a href="http://www.sigmaaldrich.com/technical-documents/articles/biology/microbial-media.html">http://www.sigmaaldrich.com/technical-documents/articles/biology/microbial-media.html</a></p>
    <p><strong>Reagents:</strong></p>
    <ul>
      <li>Tryptone (VWR 84610.0500)</li>
      <li>Yeast extract (Sigma-Aldrich Y1625-250G)</li>
      <li>NaCl Labkem (SOCH-00T-1K0)</li>
      <li>Agar powder (Sigma-Aldrich 050040-250G)</li>
      <li>Distilled water </li>
      <li>Chloramphenicol (Sigma-Aldrich C0378-25G)</li>
      <li>Ampicillin (Sigma-Aldrich A9393-5G)</li>
      <li>Kanamycin (Sigma-Aldrich 60615-5G)</li>
    </ul>
    <p><strong>Materials: </strong></p>
      <ul>
          <li>Autoclave (Nanbei LS-100LJ)</li>
          <li>Magnetic stirrer (Bioblock scientific 94361)</li>
          <li>Water purification system (Merck Millipore Elix 3)</li>
          <li>Analytical balance (Ohaus Discovery)</li>
          <li>Biosafety cabinet (ESI-Flufrance Cytair 125)</li>
          <li>Petri dishes (Monolab PDIP-10E-500)</li>
      </ul>
      <h3 class="TITRE"><strong>3. SOC Medium preparation (500ml)</strong></h3>
      <p>Mix the following component in 500mL of water :</p>
      <ul>
        <li>480 mg MgCl2</li>
        <li>0,605g MgSO4</li>
        <li>1,916g glucose</li>
        <li>95mg KCl</li>
        <li>295mg NaCl</li>
        <li>10g tryptone</li>
        <li>2,5g yeast extract</li>
      </ul>
      <p>Add a magnetic stirrer bar and mix. Name the bottles and close them with foil paper (you can also use a bottle cap). Autoclave at 121°C for 30 minutes.
         We recommend storage at 4°C but you can leave the sterile medium under the BSC.
      </p>
      <p><strong>Source: </strong><a href="https://www.qiagen.com/au/resources/faq?id=d3be05d0-ec02-4ced-aabe-dcd2a1061920&lang=en">https://www.qiagen.com/au/resources/faq?id=d3be05d0-ec02-4ced-aabe-dcd2a1061920&lang=en</a></p>
      <p><strong>Reagents:</strong></p>
      <ul>
        <li>Tryptone (VWR 84610.0500)</li>
        <li>Yeast extract (Sigma-Aldrich Y1625-250G)</li>
        <li>NaCl (Labkem SOCH-00T-1K0)</li>
        <li>Distilled water </li>
        <li>MgCl2 (Sigma-Aldrich M8266-100G)</li>
        <li>MgSO4 (Labkem MGSU-A0P-1K0)</li>
        <li>Glucose (Sigma-Aldrich G6152-1KG)</li>
        <li>KCl (Labkem POCL-00P-1K0)</li>
      </ul>
      <p><strong>Materials: </strong></p>
      <ul>
        <li>Autoclave Nanbei (LS-100LJ)</li>
        <li>Magnetic stirrer (Bioblock scientific 94361)</li>
        <li>Water purification system( Merck Millipore Elix 3)</li>
        <li>Analytical balance (Ohaus Discovery)</li>
        <li>Biosafety cabinet (ESI-Flufrance Cytair 125)</li>
      </ul>
      <h2>Protocol 2: glycerol storage</h2>
      <p><strong>Aim: </strong>safely store cells at -80°C for later uses</p>
      <p><h3 class="TITRE">1. 50% Glycerol Freezing Solution (500ml)</h3></p>
      <p>Prepare a 50% glycerol stock solution by mixing 294mL 85% glycerol solution with 206mL distilled sterile water. Mix well to make sure that you see one uniform solution, and there are no layers present.</p>
      <p><h3 class="TITRE">2. Bacteria storage</h3></p>
      <p>Mix 500µL of the overnight culture of bacteria of interest with 500µL 50% glycerol solution (25% glycerol final concentration) into a 1,8mL cryotube.</p>
      <p><strong>Source: </strong><a href="https://www.addgene.org/protocols/create-glycerol-stock/">https://www.addgene.org/protocols/create-glycerol-stock/</a></p>
      <p><strong>Reagents:</strong></p>
      <ul>
        <li>Glycerol 85% Merck Emsure</li>
        <li>Distilled water</li>
      </ul>
      <p><strong>Materials:</strong></p>
      <ul>
        <li>1,8ml cryotubes (Sigma-Aldrich V7634-500EA)</li>
        <li>Biosafety cabinet (ESI-Flufrance Cytair 125)</li>
        <li>-80°C Freezer (Froilabo)</li>
      </ul>
      <h2 class="TITRE">Protocol 3: preparation of competent cells with CaCl2 </h2>
      <p><strong>Aim :</strong>prepare competent cells for transformations</p>
      <p>Make sure to start it at the beginning of the week and have a 50ml tube centrifuge that cools down to 4°C.</p>
      <h3 class="TITRE">Day 0 : preparation </h3>
      <ul>
        <li>Streak a plate from glycerol stock. No antibiotic.</li>
        <li>Make sure all the glassware is clean, there should be NO TRACES of detergents/chemical, so make sure to rinse well with water before sterilising the glassware.</li>
        <li>Prepare and autoclave solutions, wait until they are at room temperature and store at 4°C </li>
      </ul>
      <ul>
            <li>100 mM CaCl2 solution</li>
            <li>100 mM MgCl2 solution</li>
            <li>100 mM CaCl2 ,15% glycerol (v/v) solution</li>
        </ul>
      </ul>
      <h3 class="TITRE">Day 1 : Pre-incubation</h3>
      <p>BE CAREFUL TO PUT YOUR MATERIAL AT 4°C AS SOON AS POSSIBLE.</p>
      <p>Manipulation with cold material is important because the cells will develop slower. You will indeed be asked to have an OD600 = 0,4 culture because cells in exponential phases of growth are the easiest to make competent. Manipulating in a cold environment is crucial to keep them in the exponential phase. Moreover, once competence is induced, cells that divide tend to lose the competence, so any favorable conditions of growth are to be avoided.</p>
      <ol>
        <li>Inoculate 100 mL of LB with 1 mL of the starter culture (made on day 1) in a 500 mL Erlenmeyer.</li>
        <li>Place some 50 mL falcon tubes and 1.5 mL eppendorf tubes on ice and put them in the fridge at 4°C. Turn on the centrifuge and cool the rotor to 4°C</li>
        <li>Incubate at 37°C with shaking until the cells reach early exponential phase, it takes 1- 3h until the culture becomes slightly turbid (OD600~0.4). Do not let the culture grow more than OD600 = 0.4.</li>
        <li>Aliquot into 50 mL pre chilled tubes and centrifuge 15 min at 4000 rpm in the big centrifuge at 4°C. Decelerate without brake. Discard supernatant.</li>
      </ol>
      <p>KEEP THE CELLS IN ICE ALL THE TIME AND DO NOT VORTEX</p>
      <ol start="5">
        <li>Resuspend GENTLY in 25 mL cold MgCl2 100 mM. Centrifuge cells 15 min at 4000 rpm in the big centrifuge at 4°C. Decelerate without brake. Discard supernatant. MgCL2 accomplishes the same role as CaCl2 so don’t skip this step.</li>
        <li>Resuspend GENTLY each pellet in 10 mL ice-cold CaCl2 100 mM solution. Keep resuspended cells on ice and place the box in the fridge overnight at 4 °C.</li>
      </ol>
      <h3 class="TITRE">Day 3 : conclusion</h3>
      <ol>
        <li>Make sure the centrifuge is cold and harvest the cells by centrifuging 15 min at 4000 rpm in the big centrifuge at 4°C. Decelerate without brake. Discard supernatant and GENTLY resuspend each pellet in 10 mL ice-cold 100 mM CaCl2 + 15% glycerol solution.</li>
      </ol>
      <p><i><strong>Note :</strong> glycerol alone is not sufficient to store competent cells, the CaCl2 is crucial to keep the competence.</i></p>
      <ol start="2">
        <li>Centrifuge cells 15 min at 2500 rpm 4°C. Discard supernatant and GENTLY resuspend each pellet in 2 mL ice-cold 100 mM CaCl2 + 15% glycerol solution.</li>
        <li>Make aliquots with 100 uL each. Freeze immediately at -80°C. You can use some aliquots directly for transformations but store the others as soon as possible.</li>
      </ol>
      <p>We recommend however to test the competence of the cells using the competent cell test kit provided by the iGEM. This way you will not waste your precious plasmids if the protocol failed.
         You can find the protocol here :<a href="http://parts.igem.org/Help:2017_Competent_Cell_Test_Kit.">http://parts.igem.org/Help:2017_Competent_Cell_Test_Kit.</a>
      </p>
      <p><strong>Source: </strong>This protocol has been provided by our advisor Nicolas Cornille (Glowee:<a href="www.glowee.eu">www.glowee.eu</a>). It worked every time and uses few reagents.</p>
      <p><strong>Reagents:</strong></p>
      <ul>
        <li>CaCl2 (Labkem CACH-A0P-1K0)</li>
        <li>MgCl2 (Sigma-Aldrich M8266-100G)</li>
        <li>Glycerol 50% (see protocol 2)</li>
        <li>Distilled water</li>
      </ul>
      <p><strong>Materials: </strong></p>
      <ul>
        <li>Autoclave (Nanbei LS-100LJ)</li>
        <li>Water purification system (Merck Millipore Elix 3)</li>
        <li>Biosafety cabinet (ESI-Flufrance Cytair 125)</li>
        <li>Incubator Shaker (New Brunswick Scientific Co .INC Series 25)</li>
        <li>Centrifuge (Jouan CR412)</li>
        <li>-80°C Freezer (Froilabo)</li>
      </ul>
      <h2 class="TITRE">Protocol 4: Electrophoresis</h2>
      <p><strong>Aim : </strong>analyse your DNA after any step</p>
      <h3 class="TITRE">1. 1X TAE buffer and Agarose gel preparation</h3>
      <p>To prepare 1L of 1X TAE buffer take 20ml of 50X TAE buffer in a 1L measuring cylinder and add distilled water until 1L.</p>
      <p>Otherwise the formula is the following (for a buffer of 50X) : </p>
      <table>
        <tr>
          <td>Tris-base</td>
          <td>242g</td>
        </tr>
        <tr>
          <td>Acetic acid 100%</td>
          <td>57.1ml</td>
        </tr>
        <tr>
          <td>EDTA</td>
          <td>100ml of 0,5M Sodium EDTA</td>
          <td>dH2O</td>
          <td>Qsp 1L</td>
        </tr>
      </table>
      <p>Source : <a href="https://media.addgene.org/cms/files/TAERecipe.pdf">https://media.addgene.org/cms/files/TAERecipe.pdf</a></p>
      <p>Agarose gels are made of TAE and from 0,7% to 2% of agar powder. The higher the concentration, the more the bands will be separated.</p>
      <p>Products needed:</p>
      <table>
        <tr>
          <td>TAE Buffer 1X</td>
          <td>100ml</td>
          <td>400ml</td>
        </tr>
        <tr>
          <td>Agar powder (1%)</td>
          <td>1g</td>
          <td>4g</td>
        </tr>
      </table>
      <p>TBE buffer can also be used.</p>
      <ol>
        <li>Put everything in an erlenmeyer and mix gently before heating.</li>
        <li>Heat the solution 1-3 min in microwave until the agarose is entirely dissolved, PROCEED BY HEATING AND STOPPING EVERY 30 SECOND TO AVOID BOILING (boiling means that the buffer is evaporating, and that the agarose concentration is increasing).</li>
      </ol>
      <p>USE INSULATING GLOVES TO AVOID BURNING AND BE CAREFUL OF ANY SPITING OUT OF THE ERLENMEYER. The gel must not be solid before poured into the cassette.</p>
      <ol start="3">
        <li>Let cool down until about 50°C (about 5 minutes is needed, it is ok when you can put your fingers around it without problems).</li>
        <li>Take the gel cassette and put the comb on it. Pour slowly the gel into the cassette TO AVOID BUBBLES. (If you have bubbles just take them out with a micropipette). Let it cool down either in the fridge (4°C) for 10-15 minutes or at room temperature for 20-30 min.</li>
      </ol>
      <h3 class="TITRE">2. Gel Migration</h3>
      <ol>
        <li>Add loading buffer to each of your digest samples. It is supposed to be 6X so 1µl of loading buffer for 5µl of sample.</li>
      </ol>
      <p><i><strong>Note:</strong>Loading buffer serves two purposes: 1) it provides a visible dye that helps with gel loading and will also allows you to gauge how far the gel has run while you are running your gel; and 2) it contains a high percentage of glycerol, so it increases the density of your DNA sample causing it settle to the bottom of the gel well, instead of diffusing in the buffer</i></p>
      <ol start="2">
        <li>Once solidified, place the agarose gel into the gel box (electrophoresis unit).</li>
        <li>Fill gel box with 1X TAE (or TBE) until the gel is covered.</li>
        <li>Carefully load a molecular weight ladder into the first lane of the gel.</li>
      </ol>
      <p><i><strong>Note: </strong>When loading the sample in the well, maintain positive pressure on the sample to prevent bubbles or buffer from entering the tip. Place the very top of the tip of the pipette into the buffer just above the well. Very slowly and steadily, push the sample out and watch as the sample fills the well. After all of the sample is unloaded, push the pipettor to the second stop and carefully raising the pipette straight out of the buffer.</i></p>
      <ol start="5">
        <li>Carefully load your samples into the additional wells of the gel.</li>
        <li>Run the gel at 110 V until the dye line is approximately 75-80% of the way down the gel (it takes about 1h30).</li>
      </ol>
      <p><i><strong>Note:</strong>Black is negative, red is positive. (The DNA is negatively charged and will run towards the positive electrode.)<strong> Always Run to Red.</strong></i></p>
      <p><i><strong>Note: </strong>A typical run time is about 1-1.5 hours, depending on the gel concentration and voltage.</i></p>
      <h3 class="TITRE">3. revealing the gel</h3>
      <ol>
        <li>Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.</li>
        <li>Make a bath of GelRed: 60µL of GelRed Nucleic Acid Stain 10,000X into 200 mL of Nuclease free water. Put the agarose gel in the bath and let it incubate for about 1h30. Make sure to cover the bath because GelRed is sensitive to light. Be very careful because GelRed is a CMR. If you want more distinct and visible bands, you can leave it O/N</li>
        <li>Using any device that has UV light, visualize your DNA fragments. Be careful to use an adapted mask or goggles in order to avoid the direct contact between your eyes and UV light.</li>
      </ol>
      <p><i><strong>Note: </strong>GelRed can also be incorporated directly into the gel (0,33X final) but the results we obtained with this technique were not clear. We advise however to make bath. They use more GelRed but you can re-use them 3-4 times.</i></p>
      <p><i><strong>Note: </strong>When using UV light, protect your skin by wearing safety goggles or a face shield, gloves and a lab coat.</i></p>
      <p><i><strong>Note: </strong>If you will be purifying the DNA for later use, use long-wavelength UV and expose for as little time as possible to minimize damage to the DNA.</i></p>
      <p><strong>Source: </strong><a href="https://www.addgene.org/protocols/gel-electrophoresis/">https://www.addgene.org/protocols/gel-electrophoresis/</a></p>
      <p><strong>Reagents: </strong></p>
      <ul>
        <li>Agarose  (VWR 0710-100G)</li>
        <li>Distilled water</li>
        <li>TAE buffer 50X (AMRESCO K915 1,6L)</li>
        <li>2-Log DNA Ladder (0.1-10.0 kb) (NEB N3200S)</li>
        <li>Gel loading dye purple 6X (NEB B7025S)</li>
        <li>GelRed Nucleic Acid Stain 10,000X (Biotium)</li>
      </ul>
      <p><strong>Materials: </strong></p>
      <ul>
        <li>Gel cast and combs (Biorad)</li>
        <li>Electrophoresis cuve (Biorad Sub-cell GT)</li>
        <li>Generator (Prolabo gen-apex 600-800)</li>
        <li>Microwave</li>
        <li>Erlenmeyer</li>
        <li>Safety insulating gloves</li>
        <li>UV Table</li>
        <li>Safety goggles</li>
      </ul>
      <h2 class="TITRE">Protocol 5 : Bacterial transformation</h2>
      <p><strong>Aim: </strong>incorporate plasmids in competent bacteria</p>
      <p><i><strong>Note: </strong>we usually used the iGEM transformation protocol but had to adapt it to our lab conditions and experience.</i></p>
      <ol>
        <li>Pipet 5µl of DNA from the then 50 µL of competent cells into each tube. Flick the tube gently with your finger to mix.</li>
      </ol>
      <p><i><strong>Note: </strong>When using highly concentrated minipreps, we used 5µl but you can use less. When using DNA from iGEM plates, depending on the competence of your cells, we recommend using 3µl instead of 1µl. When using ligation products, put the 50µl of bacteria directly with the 20µl of ligation.</i></p>
      <p>We also did a blank with 50µl of competent cells and 1µl of water. This is to measure if the cells survived the storage.</p>
      <ol start="2">
        <li>Incubate on ice for 30 minutes. Put it in the fridge, the temperatures in the room are too high (it was the case in our lab).</li>
        <li>Pre-heat waterbath now to 42°C. </li>
        <li>Heat-shock the cells by placing into the waterbath for 30 seconds (no longer than 1 min). Be careful to keep the lids of the tubes above the water level, and keep the ice close by. 30 seconds worked everytime for us so we recommend this time.</li>
        <li>Immediately transfer the tubes back to ice, and incubate on ice for 5 minutes. IN THE FRIDGE AS WELL</li>
        <li>Add qsp 1ml of SOC medium WITHOUT CHLORAMPHENICOL per tube, and incubate at 37°C for 1 hour 30min shaking at 200-300 rpm.</li>
      </ol>
      <p><i><strong>Note: </strong>we once left the incubation to 4h. This generated more cells but we do not recommend letting more than 1h30 because it can favor growth of unwished clones and make bacterial selection on petri dishes very difficult.</i></p>
      <ol start="7">
        <li>Prepare the agar plates during this time: label them. Put the agar plates at 37°C.</li>
        <li>Tubes were centrifuged at 3500rpm for 4minutes and 850µL of supernatant was discarded. Bacteria were resuspended into the remaining culture media and were spread in the corresponding petri dishes.</li>
      </ol>
      <p>This step worked fine for us because our ligation product was not very concentrated and that way we ensured to have all the cells possible to inoculate in the selection plates.</p>
      <p>Put the dishes in the incubator (37°C advised).</p>
      <p><strong>Source: </strong><a href="http://parts.igem.org/Help:Protocols/Transformation">http://parts.igem.org/Help:Protocols/Transformation</a></p>
      <p><strong>Reagents: </strong></p>
      <ul>
        <li>Ligated plasmid</li>
        <li>Competent cells (DH5-alpha, see protocol 3)</li>
        <li>SOC medium (see protocol 1)</li>
      </ul>
      <p><strong>Material: </strong></p>
      <ul>
        <li>Inoculating loop</li>
        <li>Incubator (Memmert B50)</li>
        <li>Centrifuge (Eppendorf 5417)</li>
        <li>Ice maker (optic ivymen system)</li>
        <li>Waterbath (Julabo 5M)</li>
        <li>Biosafety cabinet (ESI-Flufrance Cytair 125)</li>
        <li>Petri dishes (Monolab PDIP-10E-500)</li>
        <li>Fridge</li>
      </ul>
      <h2 class="TITRE">Protocol 6 : DNA resuspension from iGEM plates</h2>
      <p><strong>Aim: </strong>solubilise DNA in solution for further experiments.</p>
      <p>To locate the part on the plate: <a href="http://igem.org/Videos/Locating_Your_Part">http://igem.org/Videos/Locating_Your_Part</a></p>
      <p>To use the DNA in the Distribution Kit, follow these instructions:</p>
      <p><i><strong>Note: </strong>There is an estimated 2-3ng of DNA in each well, following this protocol, assume that you are transforming with 200-300pg/µL.</i></p>
      <ol>
        <li>With a pipette tip, punch a hole through the foil cover into the corresponding well of the part that you want. Make sure you have properly oriented the plate. Do not remove the foil cover, as it could lead to cross contamination between the wells.</li>
        <li>Pipette 10µL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully resuspended. The resuspension will be red, as the dried DNA has cresol red dye.</li>
      </ol>
      <p><i><strong>Note: </strong>We recommend that you do not use TE to resuspend the dried DNA.</i></p>
      <ol start="3">
        <li>Transform 1µL of the resuspended DNA into your desired competent cells, plate your transformation with the appropriate antibiotic and grow overnight.</li>
        <li>Pick a single colony and inoculate broth (again, with the correct antibiotic) and grow for 16 hours.</li>
      </ol>
      <p><strong>Source: </strong><a href="http://parts.igem.org/Help:2017_DNA_Distribution#DNA_Kit_Plate_Instructions">http://parts.igem.org/Help:2017_DNA_Distribution#DNA_Kit_Plate_Instructions</a></p>
      <p><strong>Reagents: </strong></p>
      <ul>
        <li>Nuclease free water</li>
        <li>DNA from iGEM plates</li>
        <li>Competent cells (DH5-alpha, see protocol 3)</li>
        <li>SOC Medium (see protocol 1)</li>
      </ul>
      <p><strong>Materials: </strong></p>
      <ul>
        <li>Biosafety cabinet ESI-Flufrance Cytair 125</li>
        <li>Petri dishes (Monolab PDIP-10E-500)</li>
        <li>Inoculating loop</li>
        <li>Incubator (Memmert B50)</li>
        <li>Centrifuge (Eppendorf 5417)</li>
        <li>Ice maker (optic ivymen system)</li>
        <li>Waterbath (Julabo 5M)</li>
        <li>Biosafety cabinet (ESI-Flufrance Cytair 125)</li>
        <li>Petri dishes (Monolab PDIP-10E-500)</li>
      </ul>
      <h2 class="TITRE">Protocol 7: PCR of DNA fragments</h2>
      <p><strong>Aim: </strong>Amplificating your DNA fragment and biobricks for later use.</p>
      <p><i><strong>Note: </strong>during this iGEM we used Q5 master mix from NEB for the first months but had later to rely on Taq polymerase for the PCR. Be careful to adapt the protocol as Taq polymerase is believed to make more errors. Make sure you know your DNA fragments length before any PCR and gel migration as this is crucial. Make sure to use nuclease-free water</i></p>
      <h3 class="TITRE">1. PCR mix preparation</h3>
      <h4 class="TITRE">With Q5 master mix from NEB</h4>
      <p>Pre-dilutions préparation:</p>
      <p>The primers used were always from IDT, the IDT tubes are at a concentration of 100µM.
         Primers 100µM: 1 µL primer + 9 µL dH2O --> 10µM final
      </p>
      <table>
        <tr>
          <td>Tubes</td>
          <td>Primer Prefix (10µM)</td>
          <td>Primer Suffix (10µM)</td>
          <td>DNA template</td>
          <td>Q5 Mix</td>
          <td>dH2O DNase/RNase free</td>
          <td>Total Volume</td>
        </tr>
        <tr>
          <td>DNA fragment</td>
          <td>5µL</td>
          <td>5µL</td>
          <td>2µL</td>
          <td>25µL</td>
          <td>13µL</td>
          <td>50µL</td>
        </tr>
      </table>
      <h4 class="TITRE">With Taq polymerase</h4>
      <p>Pre-dilution Primers 100µM: 1,2 µL primer + 10,8 µL dH2O --> 10µM final, volume total final (12µl-->we need 10ul)</p>
      <p>Pre-dilution Taq Polymerase 5000U/mL: 1,2µL Taq + 10,8µL dH2O --> 500U/mL final, volume total final (12µl-->we need 10µl)</p>
      For a 50µl PCR mix volume:
      <table>
        <tr>
          <td>Components</td>
          <td>Volume</td>
          <td>Final volume</td>
        </tr>
        <tr>
          <td>10X Standard Taq Reaction Buffer</td>
          <td>5µL</td>
          <td>1X (1/10)</td>
        </tr>
        <tr>
          <td>10mM dNTPs</td>
          <td>1µL</td>
          <td>200µM (1/50)</td>
        </tr>
        <tr>
          <td>10µM Prefix Primer</td>
          <td>5µL</td>
          <td>1µM</td>
        </tr>
        <tr>
          <td>10µM Suffix Primer</td>
          <td>5µL</td>
          <td>1µM</td>
        </tr>
        <tr>
          <td>Taq DNA Polymerase 500U/mL</td>
          <td>5µL</td>
          <td>2,5 units/50µL PCR</td>
        </tr>
        <tr>
          <td>DNA template (10ng/µL)</td>
          <td>2µL</td>
          <td></td>
        </tr>
        <tr>
          <td>Nuclease-free water</td>
          <td>27µL</td>
          <td>qsp 50µL</td>
        </tr>
      </table>
      <p>We divided our 50µL mix into two 25µl mixes. The PCR is then performed with only 18 cycles to reduce the risk of errors (Taq polymerase not very accurate). Finally the two PCR products are mixed together before the clean up. The division by two was advised because of the higher error rate from Taq. That way we put two times 18 cycles instead of one time 30 cycle and lower the error rate.</p>
      <h3 class="TITRE">2. PCR cycle </h3>
      <p>PCR thermocycler program:</p>
      <table>
        <tr>
          <td>PCR Cycle</td>
        </tr>
        <tr>
          <td>Repetition</td>
          <td>Step</td>
          <td>Time</td>
          <td>Temperature</td>
        </tr>
        <tr>
          <td>0</td>
          <td>Initial denaturation</td>
          <td>30 sec</td>
          <td>95°C</td>
        </tr>
        <tr>
          <td>30</td>
          <td>Denaturation</td>
          <td>30 sec</td>
          <td>95°C</td>
        </tr>
        <tr>
          <td> </td>
          <td>Annealing</td>
          <td>1 min</td>
          <td>Tm*</td>
        </tr>
        <tr>
           <td> </td>
           <td>DNA polymerization</td>
           <td>(30 sec/kb)*</td>
           <td>68°C</td>
        </tr>
        <tr>
          <td>0</td>
          <td>DNA polymerization</td>
          <td>5 min</td>
          <td>68°C</td>
        </tr>
        <tr>
          <td> </td>
          <td>FINISH</td>
          <td>∞</td>
          <td>4°C</td>
        </tr>
      </table>
      <p>*Remember to adapt the polymerisation step according to your DNA fragment length. Also adapt the annealing temperature according to your primers.</p>
      <p>You can also store your samples at -20°C for longer uses.</p>
      <h3 class="TITRE">3. Gel Migration</h3>
      <p>We use 2/4µl of the PCR products to quantify the sequences concentrations efficiently on an agarose gel.</p>
      <p>Mixes for deposit:</p>
      <p>7µL PCR product + 1,4µL 6X loading dye → 8,4µL total</p>
      <p>2µl DNA: 2,4µl of the mix</p>
      <p>4µl DNA: 4,8µl of the mix</p>
      <p>This is advised for ligation step since you need to know your DNA concentration. However, if you have a nanodrop, just migrate 2µl once to know if you have the appropriate fragment in your tube and then use the nanodrop to have a better estimation of your DNA quantity.</p>
      <p><i><strong>Note: </strong>we recommend digesting the backbone you are going to use for the ligation of your sequence and make 2/4/8 digestion product migrated at the same time as your PCR samples.</i></p>
      <p>That way you will need one gel only to quantify both you insert and backbone.
         Ignore this advice if you do a migration of your digested inserts, since you will need two gels anyway.
      </p>
      <p>If the PCR failed, you might want to check this link to spot the cause : <a href="https://www.neb.com/tools-and-resources/troubleshooting-guides/pcr-troubleshooting-guide.">https://www.neb.com/tools-and-resources/troubleshooting-guides/pcr-troubleshooting-guide.</a></p>
      <p>We had some troubles and it was all the time because of our primers so be very careful.</p>
      <p><strong>Source: </strong><a href="https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491">https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491</a></p>
      <p><strong>Reagents: </strong></p>
      <ul>
        <li>Forward primer (100µM) synthesized by IDT</li>
        <li>Reverse primer (100µM) synthesized by IDT</li>
        <li>DNA template (synthesized by IDT)</li>
        <li>Q5® High-Fidelity 2X Master Mix (NEB)</li>
        <li>Taq polymerase and buffer (NEB M0273S)</li>
        <li>DNTP mix (VWR 733-1363)</li>
        <li>DNase/RNase free water</li>
      </ul>
      <p><strong>Materials: </strong></p>
      <ul>
        <li>Temperature gradients thermocycler (Eppendorf Mastercycler gradient)</li>
        <li>Thermocycler (Eppendorf Mastercycler)</li>
      </ul>


      <h2 class="display-3">Protocol 8 : PCR product purification</h2>
        <p class= "txt"><strong>Aim</strong>: harvest your PCR product and get rid of the reagents such as DNA polymerase, buffers and DNTPs.</p>
        <p class= "txt">Note: take into account that a PCR clean-up induces an average loss of 25% of your PCR product, but is necessary.</p>
        <ol>
          <p class= "txt"><li> All these steps are performed simultaneously for each PCR product. One tubes of each sequences were used, so the second one for each remains at -20°C.</p>
          <p class= "txt">CAUTION: all the steps must be performed using DNase-free materials (tips, gloves, water).</p>

            <li> Add 5 volumes (200µL) Buffer PB to 1 volume of the PCR reaction (40µL) and mix. The color of the mixture will turn yellow. (If the color of the mixture is orange or violet, add 10µL 3M sodium acetate, pH 5,0, and mix.</p>
            <li> Preparation of sodium acetate 3M: 2,46 mg in 10µL distilled water). CAUTION: cut the edge of the tip before collecting the Buffer PB, and only collect small volumes (2x 100µL), as it is highly viscous.</p>
            <li> Place a QIAquick column in a provided 2mL collection tube.</p>
            <li> To bind DNA, apply the sample to the QIAquick column and centrifuge for 1min at 13000rpm.</p>
            <li> Discard flow-through and place the QIAquick column back in the same tube.</p>
            <li> To wash, add 750µL Buffer PE to the QIAquick column, centrifuge for 1min at 13000rpm. </p>
            <li> Discard flow-through and place the QIAquick column back in the same tube.</p>
            <li> Centrifuge the QIAquick column once more in the provided 2mL collection tube for 1min to remove residual wash buffer.</p>
            <li> Place each QIAquick column in a clean 1.5mL microcentrifuge tube.</p>
            <li> To elute DNA, add 50µL DNase-free water to the center of the QIAquick membrane and centrifuge the column for 1min at 13000rpm. You can also put 25µl of water, centrifuge for 1 min at 13000 rpm; then repeat the step (this is advised).</p>
            <li> Repeat the previous step without discarding the flow-through for increased DNA concentration.</p>
          Note: If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel. We generally did the PCR clean-up after the migration to not wasted any reagent in case there were no PCR fragments and the PCR failed.</p>
            <li>DNA samples were stored at -20°C. From here you can either send the samples for sequencing or carry on for the digestion and ligation.</p>
        </ol>

        <p class= "txt">Source: <a href="https://www.qiagen.com/fr/shop/sample-technologies/dna/dna-clean-up/qiaquick-pcr-purification-kit/#orderinginformation">https://www.qiagen.com/fr/shop/sample-technologies/dna/dna-clean-up/qiaquick-pcr-purification-kit/#orderinginformation<a></p>
        <p class= "txt"><strong>Reagents:</strong></br>
          <ul>
            <li>QIAquick PCR Purification Kit (Quiagen 28104)</li>
            <li> Distilled water</li>
            <li>Ethanol 96%</li>
            <li>PCR fragments</li>
            <li>Sodium Acetate 3M</li>
          </ul>
          <strong>Materials:</strong></br>
          <ul>
            <li>Centrifuge (Eppendorf 5417)</li>
          </ul></p>




        <h2 class="display-3">Protocol 9 : Enzyme digestion</h2>
        <p class= "txt">For standard biobrick - backbone ligation, digestion needs to be performed with EcorI and PstI for both backbone and insert. For biobrick assembly, digestion needs to be performed with EcorI and XbaI for the backbone+part 1 and EcorI and PstI for the part 2.
          Source :<a href="http://parts.igem.org/Assembly:Standard_assembly">http://parts.igem.org/Assembly:Standard_assembly</a></p>
        <p class= "txt">Be careful to select your digestion buffer according to the two enzymes you are using. Same goes for the heat inactivation. All the informations are provided by the NEB website.
  The final volume of each digestion is 50µl.</p>

        <table>
          <tr>
            <td>Reagents</td>
            <td>Quantity</td>
            <td>Final concentration</td>
          </tr>
          <tr>
            <td>Enzyme 1</td>
            <td>1 µL</td>
            <td>40 U/µL</td>
          </tr>
          <tr>
            <td>Enzyme 2</td>
            <td>1 µL</td>
            <td>40 U/µL</td>
          </tr>
          <tr>
            <td>Purified sequence</td>
            <td>30 µL</td>
            <td>10 ng/µL</td>
          </tr>
          <tr>
            <td>10X NEB Buffer Y</td>
            <td>5 µL</td>
            <td>1X</td>
          </tr>
          <tr>
            <td>Ultrapure water</td>
            <td>13 µL</td>
            <td> </td>
          </tr>
        </table>

        <p class= "txt">Digestion was carried out 2 hours at 37°C. This is overall very long but we adapted the incubation time to be certain that the digestion was carried out.
  Heat inactivation was proceeded at 80°C for 20 minutes.</p>
        As for the PCR, we recommend migration of digested samples to make sure the process worked. We did this a lot at the beginning. You can also run a gel to extract the digested backbone. </br>
        Backbones used were generally PSB1C3-mRFP (BBa_J04450) or PSB1C3-seq 2 (BBa_K2282006) for us, so we had first to remove the insert to replace it by our sequence.
  When we wanted to put an insert that expresses AmilCP, we used a backbone containing originally mRFP.</br>
        When we wanted to put an insert that expresses mRFP, we used a backbone containing originally AmilCP.</br>
        That way we did not need either colony PCR or gel extraction because we just had to select the colonies that had the right colors (in our plates there were red, white and blue colonies).
  PCR colony was necessary when we needed to select white colonies.</br>

        <p class= "txt"><strong>Source:</strong> <a href="https://www.neb.com/protocols/0001/01/01/digestion-protocol-e0546">https://www.neb.com/protocols/0001/01/01/digestion-protocol-e0546</a></p>
        <p class= "txt"><strong>Reagents:</strong>
          <ul>
            <li> Plasmid pSB1C3-mRFP (psB1C3-BBa_J04450 from the IGEM Distribution Kit)</li>
            <li> Biobrick Assembly kit (NEB E0546S)</li>
          </ul>
          <strong>Materials:</strong>
          <ul>
            <li>Ultrapure water</li>
            <li>Waterbath (Julabo 5M)</li>
            <li>Ice maker (optic ivymen system)</li>
          </ul>
        </p>




      <h2 class="display-3">Protocol 10 : Ligation</h2>
      <p class= "txt"><strong>Aim:</strong> ligate insert inside backbone to create a plasmid</p>
      For this step make sure you know the concentration of both your insert and backbone by using either the gel migration results or a nanodrop. We tried 1:1, 1:3 and 1:5 ratio and the best seemed to be 1:3 and 1:5 (one mole of backbone : three/five moles of inserts).
  For the ratio, we used the NEB website : <a href="https://nebiocalculator.neb.com/#!/ligation">https://nebiocalculator.neb.com/#!/ligation</a></p>
      Remember : if you quantify your PCR product on gel like us, take in account that a PCR purification makes on average lose 25% of your product. As you digest your whole sample, no loss is to be taken from the digestion step. Any purification step loses material so be careful not to overdo it (even though it is better).</p>

      <table>
        <tr>
          <td>Insert</td>
          <td>X µL (depending on concentration and ratio)</td>
        </tr>
        <tr>
          <td>Backbone</td>
          <td>Y µL (depending on concentration and ratio)</td>
        </tr>
        <tr>
          <td>10X T4 DNA Ligase Buffer</td>
          <td>2 µL</td>
        </tr>
        <tr>
          <td>T4 DNA Ligase</td>
          <td>1 µL</td>
        </tr>
        <tr>
          <td>dH2O</td>
          <td>6,1 µL</td>
        </tr>
        <tr>
          <td>Total volume</td>
          <td>20 µL</td>
        </tr>
      </table>

      <p class= "txt">Incubate at 16°C O/N or 15 min at room temperature.
  We used the 16°C incubation all the time and we recommend it. However make sure to transform quickly with the ligation product once it is finished, even storing can degrade it.
  Remember to transform with the whole ligation product (20µl).</br>
      <strong>Source :</strong> <a href="https://www.neb.com/protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202">https://www.neb.com/protocols/0001/01/01/dna-ligation-with-t4-dna-ligase-m0202</a></p>

    <p class= "txt"><strong>Reagents:</strong>
      <ul>
        <li>Digested insert</li>
        <li>Digested backbone</li>
        <li>10X T4 DNA Ligase Buffer (NEB M0202S)</li>
        <li>T4 DNA ligase (400000U/mL NEB M0202S)</li>
        <li>DNase free water</li>
      </ul>
        <strong>Materials:</strong>
      <ul>
        <li>Ice maker (optic ivymen system)</li>
        <li>Water purification system (Merck Millipore Elix 3)</li>
      </ul>




      <h2 class="display-3">Protocol 11 : PCR Colony </h2>
      <p class= "txt">This step is very useful to make sure you have the correct plasmid in a chosen colony before any culture for further minipreps and glycerol stocks. We did not use it when we had color reporters (AmilCP and mRFP), but when no color reporter was used (for our sequence 7 or the protegrin for example), we relied on PCR colony.</br>
      Make sure you know the length of your insert+backbone package.</p>
      As you will need a lot of PCR mix, we recommend preparing a big PCR mix that you can split up into all the tubes.</p>
      Choose the colonies that you want to analyse. Each chosen colony has been annotated on the petri dishes so we will be able to use them again for liquid culture later. Make sure to store the petri dishes at 4°C.</p>

      <p class= "txt"><strong>1) Cell membrane destruction :</strong></p>
      In each PCR tube: put 8µL of dH20. Pick a colony (with a toothpick or a micropipette tip), add a stab of it into the PCR tube. Do not take the whole colony on the dish because you will need it after the analysis if it carries the right plasmid !
      Incubate the tubes at 95°C for 5min.</p>
      <strong>2) Prepare the giant PCR mix:</strong></p>
      We recommend using a control by doing a mix for a known plasmid in miniprep, like PSB1C3-mRFP (BBa_J04450). You of course don’t need the previous step for the plasmid.</p>

      <table>
        <tr>
          <td>Components</td>
          <td>Volume</td>
          <td>Final concentration</td>
        </tr>
        <tr>
          <td>10X Standard Taq Reaction Buffer</td>
          <td>17,5 µL</td>
          <td>1X</td>
        </tr>
        <tr>
          <td>10mM dNTPs</td>
          <td>3,8 µL</td>
          <td>200 µM (1/50) µL</td>
        </tr>
        <tr>
          <td>10 µM Prefix Primer</td>
          <td>1,75 µL</td>
          <td>1 µM</td>
        </tr>
        <tr>
          <td>10 µM Suffix Primer</td>
          <td>1,75 µL</td>
          <td>1 µM</td>
        </tr>
        <tr>
          <td>Taq DNA Polymerase</td>
          <td>1,75 µL</td>
          <td>2,5 units/50 µL PCR
        </tr>
        <tr>
          <td>Nuclease-free water</td>
          <td>148,75 µL</td>
          <td>qsp 175 µL</td>
      </table>

      Put 17µL of mix in each tube (25-8=17). Remember that there is already 8µl of water and your destroyed cells in each tube, so the final volume will be 25µl</p>
      <strong>3) Cycles :</strong></p>
      As for the PCR, adapt annealing and polymerisation steps according to your primers and sequence length.</p>
      PCR thermocycler program:</p>

      <table>
        <tr>
          <td>PCR Cycle</td>
        </tr>
        <tr>
          <td>Repetition</td>
          <td>Step</td>
          <td>Time</td>
          <td>Temperature</td>
        </tr>
        <tr>
          <td>0</td>
          <td>Initial denaturation</td>
          <td>30 sec</td>
          <td>95°C</td>
        </tr>
        <tr>
          <td>30</td>
          <td>Denaturation</td>
          <td>30 sec</td>
          <td>95°C</td>
        </tr>
        <tr>
          <td> </td>
          <td>Annealing</td>
          <td>1 min</td>
          <td>60°C</td>
        </tr>
        <tr>
          <td> </td>
          <td>DNA polymerization</td>
          <td>30 sec/kb</td>
          <td>68°C</td>
        </tr>
        <tr>
          <td>0</td>
          <td>DNA polymerization</td>
          <td>5 min</td>
          <td>68°C</td>
        </tr>
          <td> </td>
          <td> FINISH</td>
          <td>∞</td>
          <td>4°C</td>
      </table>

      Note that we use 30 cycles with the Taq polymerase as opposed to the 18 cycles for the PCR. PCR colony being only for analysis, mutation caused by errors of the Taq are not problematic. Taq is better for PCR colony because it is cheaper than Q5 master mix and you don’t need error-free polymerase.</p>

      <strong>4) Results:</strong></p>
        Load 10µL of the PCR mix + 2µL of 6X loading dye into agarose gels and migrate/analyse with UV.</p>

      <strong>Source :</strong>This protocol has been provided by our advisor NIcolas Cornille from Glowee (<a href="http://www.glowee.eu/">http://www.glowee.eu/</a> )</p>
      <strong> Reagents:</strong></br>
        <ul>
          <li>Taq polymerase and buffer (NEB M0273S)</li>
          <li>10µM dNTPS (NEB)</li>
          <li>Forward primer 100µM (synthesized by IDT)</li>
          <li>Reverse primer 100µM (synthesized by IDT)</li>
        </ul>
      <strong>Materials</strong></br>
        <ul>
          <li>DNAse/ RNAse free water</li>
          <li>PCR tubes</li>
          <li>Thermocycler</li>
          <li>Micropipettes and tips</li>
          <li>DNA migration material (see protocol 4)</li>
          <li>Ice maker (optic ivymen system)</li>
        </ul>




        <h2 class="display-3">Protocol 12 : Liquid culture of transformed bacteria for plasmid amplification.</h2>
        <p class= "txt"><strong>Aim:</strong> make bacteria grow and amplify plasmid</p>
        Once your transformation with ligation product  and PCR colony achieved, it is crucial to cultivate the right colonies in liquid medium to amplify the plasmid and make glycerol stocks and miniprep.</p>

        <ol>
          <p class= "txt">
          <li>Prepare 10ml of liquid LB medium per colony with corresponding antibiotics into 50ml sterile falcon tubes (it is better to use culture tubes but we did not have access to them, falcon are working fine).</p>
          <li>Pick a single colony that has the right plasmid with a sterile loop of a sterile pipet tip. (you can choose with color reporters or PCR colony results (that is why you do not take a whole colony while doing PCR colony).</p>
          <li>Put the tip/loop into the corresponding falcon tube and gently shake to free the attached bacteria. If you are using a tip you can just leave the tip into the falcon tube (if it is sterile).</p>
          <li>Put the falcon tubes at 37°C at 250 rpm. Make sure to slightly open the cap to let oxygen pass. Leave O/N (16-24h). After incubation the medium should be turbid and the bacteria ready for glycerol stocks and miniprep.</p>
          Note: E.coli is a facultative anaerobic bacteria (that makes ATP by aerobic respiration if oxygen is present but is capable of switching to fermentation or anaerobic respiration if oxygen is absent) but we observed better chromoproteins production in presence of oxygen so be sure to let the cap slightly opened.</p>
        </ol>
        <strong>Reagents:</strong>
          <ul>
            <li>Liquid LB with antibiotics (see protocol 1)</li>
            <li>Correctly selected colony</li>
          </ul>
        <strong>Materials:</strong>
          <ul>
            <li>Incubator Shaker (New Brunswick Scientific Co .INC Series 25)</li>
            <li>Sterile inoculating loop</li>
            <li>Sterile falcon tube 50ml  (Falcon 352098)</li>
          </ul>




          <h2 class="display-3">Protocol 13: Miniprep of plasmids</h2>
          <p class= "txt"><strong>Aim:</strong> extract plasmids from transformed bacteria</p>
          This protocol uses the Monarch miniprep kit from NEB but we adapted the protocol for some steps. Follow the link at the end to have the original.</br>
          Make sure to use DNAse free water for the final step instead of Elution Buffer, this has been advised to us because it is better for further experiments. Elution Buffer provided by the kit is although useful for long-term storage.</p>

          All centrifugation steps should be carried out at 16,000 x g (~13,000 RPM).</br>
          If precipitate has formed in Lysis Buffer (B2), incubate at 30–37°C, inverting periodically to dissolve.</br>
          Store Plasmid Neutralization Buffer (B3) at 4°C after opening.<br>
          <strong>Note:</strong> unlike other commercial kits, all wash steps are required.</p>

          <ol>
            <li>Pellet 3 mL bacterial culture by centrifugation for 1 minute. Discard supernatant.</p>
            <li>Resuspend pellet in 200 μl Plasmid Resuspension Buffer (B1) (pink). Vortex or pipet to ensure cells are completely resuspended. There should be no visible clumps.</p>
            <li>Lyse cells by adding 200 μl Plasmid Lysis Buffer (B2) (blue). Invert tube immediately and gently 5–6 times until color changes to dark pink and the solution is clear and viscous. Do not vortex! Incubate for 1 minute at room temperature.</br>
              Note: Care should be taken not to handle the sample roughly and risk shearing chromosomal DNA, which will co-purify as a contaminant. Avoid incubating longer than one minute to prevent irreversible plasmid denaturation.</p>
            <li>Neutralize the lysate by adding 400 μl of Plasmid Neutralization Buffer (B3) (yellow). Gently invert tube until color is uniformly yellow and a precipitate forms. Do not vortex! Incubate at room temperature for 2 minutes.<br>
              Note: Be careful not to shear chromosomal DNA by vortexing or vigorous shaking. Firmly inverting the tube promotes good mixing, important for full neutralization.</p>
            <li>Clarify the lysate by spinning for 2-5 minutes at 16,000 x g.</br>
              Note: Spin time should not be less than 2 minutes. Careful handling of the tube will ensure no debris is transferred and the 2 minute recommended spin can be successfully employed to save valuable time. For culture volumes > 1 ml, we recommend a 5 minute spin to ensure efficient RNA removal by RNase. Also, longer spin times will result in a more compact pellet that lower the risk of clogging the column.</p>
            <li>Carefully transfer supernatant to the spin column and centrifuge for 1 minute. Discard flow-through. To save time, spin for 30 seconds, instead of 1 minute.</p>
            <li>Re-insert column in the collection tube and add 200 μl of Plasmid Wash Buffer 1. Plasmid Wash Buffer 1 removes RNA, protein and endotoxin. (Add a 5 minutes incubation step before centrifugation if the DNA will be used in transfection.) Centrifuge for 1 minute. Discarding the flow-through is optional.</br>
              Note: The collection tube is designed to hold 800 μl of flow-through fluid and still allow the tip of the column to be safely above the top of the liquid. Empty the tube whenever necessary to ensure the column tip and flow-though do not make contact.</p>
            <li>Add 400 μl of Plasmid Wash Buffer 2 and centrifuge for 1 minute. Allow the solution to pass through the column, then switch the vacuum source off.</p>
            <li>Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column has not come into contact with the flow-through. If there is any doubt, re-spin the column for 1 minute before inserting it into the clean microfuge tube.</p>
            <li>Add 30µl of Nuclease free water to the center of the matrix. Wait for 1 minute, then spin for 1 minute to elute DNA. You can repeat the step to have 60µl of final volume. You can also do two times 15µl to have 30µl of final volume.</p>
          </ol>

          <p class= "txt"><strong>Source:</strong> <a href="https://www.neb.com/protocols/2015/11/20/monarch-plasmid-dna-miniprep-kit-protocol-t1010">https://www.neb.com/protocols/2015/11/20/monarch-plasmid-dna-miniprep-kit-protocol-t1010</a></p>
          <p class= "txt"><strong>Reagents :</strong>
            <ul>
              <li>Monarch® Plasmid Miniprep Kit (NEB T1010S)
            </ul>
          <p class= "txt"><strong>Materials :</strong>
            <ul>
              <li>Centrifuge (Eppendorf 5417)</li>
              <li>Ice maker (optic ivymen system)</li>
            </ul>



          <h2 class="display-3">Protocol 14 : Protein extraction</h2>
          <p class= "txt">This protocol is inspired by an article and adapted to our laboratory. It did not provide any great results and consumed a lot of material so we do not advise other groups to use it. We post it here anyway.</p>
          Stock: SDS (10%) stock solution</br>
          Dissolve 10 g of SDS in 80 mL of H2O, and then add H2O to 100 mL. This stock solution is stable for 6 months at room temperature. </br>
          (<a href="http://cshprotocols.cshlp.org/content/2006/1/pdb.rec10480.full?text_only=true">http://cshprotocols.cshlp.org/content/2006/1/pdb.rec10480.full?text_only=true</a>)</p>

          <ol>
            <li>Cells from 3 ml of culture were harvested by centrifugation (13000 rpm/1min), washed twice with phosphate-buffered saline without Mg2+ and Ca2+  and re centrifuged (13000 rpm/1min).
            Note: For this step you can centrifuge 1ml, discard the supernatant, then add another 1 ml and re-centrifuge, discard again, and add a final 1 ml and centrifuge again, and discard again.</p>
            <li>The cells were then suspended in 1,5 ml of ice-cold acetone, allowed to stand in fridge for 5 min, and collected by centrifugation (13000 rpm 1min). Acetone was taken off by evaporation on heating plates (60°C). Do not let the heating proceed too long, otherwise the pellet might become too hard to resuspend.</p>
            <li>Proteins were then extracted by incubating and homogenizing (pipetting, vortexing) the pellet with 150µl of 1% sodium dodecyl sulfate (SDS) for 2 min (13000 rpm/1min). Be careful of the pellet.</p>
            <li>Harvest the supernatant, complete with 1350 µl of distilled water and analyse. be careful of the very unstable pellet.
          </ol>

          <p class= "txt"><strong>Source:</strong>Bhaduri et al., “Simple and Rapid Method for Disruption of Bacteria for Protein Studies”, APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Oct. 1983, p. 941-943</p>
          <strong>Reagents:</strong>
            <ul>
              <li>Acetone (Onyx Bricolage ME670898)</li>
              <li>SDS 20% (Biosolve 19812323)</li>
              <li>PBS 10X (Volusol 87479290)</li>
              <li>Distilled water</li>
              <li>Bacterial culture</li>
            </ul>
          <strong>Reagents:</strong>
            <ul>
              <li>Spectrometer (Beckman DU530)</li>
              <li>Plastic cuve (Sarstedt 67.745)</li>
              <li>Ice maker (optic ivymen system)</li>
            </ul>



          <h2 class="display-3">Useful informations for OD measurements :</h2>
          <p class= "txt">AmilCP</br>
            Maximum of absorption : 588 nm</br>
            Molar extinction coefficient : 87600</br>
            Source : Alieva et al., “Diversity and Evolution of Coral Fluorescent Proteins”, Published: July 16, 2008 <a href="https://doi.org/10.1371/journal.pone.0002680">https://doi.org/10.1371/journal.pone.0002680</a>, PLOS</p>

            <p class= "txt">mRFP</br>
              Maximum of absorption : 584 nm</br>
              Emission : 607 nm</br>
              Molar extinction coeff : 32890</br>
              Source: <a href="http://parts.igem.org/Part:BBa_E1010">http://parts.igem.org/Part:BBa_E1010</a></p>

            <p class= "txt">For E. coli, the OD600 of 1.0 = 8e+8 cells/ml</p>




            <h2 class="display-3">Protocol 15 : Cold-response characterisation through colorimetry</h2>
            <p class= "txt"><strong>Aim:</strong> evaluate the bacteria growth and protein production at low temperature</p>
              This is our protocol for the Cold-Response characterisation and the evaluation of our sequences 1 (BBa_K2282005), 2 (BBa_K2282006) and 7 (BBa_K2282011). After many trials with protein extraction that failed, we decided to make it more simple and to use colorimetry, with our color reporters, to assess if it worked.</br>
              Make sure to have two sterile erlenmeyers per culture to be characterised, as we incubate at two different temperatures.</br>
              For each transformed culture that needs to be characterised :</p>

              <p class= "txt"><strong>Day 0 : Pre-Incubation</strong></p>
              As for the competent cell protocol, an OD600 kinetic will be made and you will need the bacteria to reach OD600 = 0,5. To this end making a pre-culture is important. It permits a time reduction for the culture to reach the OD600 (warm-up). We had many troubles reaching the right OD600 without the warm-up so we advise you to do this step.</p>

              <ol>
                <li>Incubate 100µl of transformed bacteria in glycerol stock in 15ml sterile falcons containing 3ml of liquid LB with corresponding antibiotics. </p>
                <li>Put the falcons at 37°C 250 rpm O/N</p>
              </ol>

              <p class= "txt"><strong>Day 1 : OD600 measurement and launch of characterisation</strong></p>
              <ol>
                <li>Put one incubator at desired cold temperature to make it ready for the incubation step.</p>
                <li>Prepare one erlenmeyer per O/N culture and put 50ml of liquid LB with corresponding antibiotics in each</p>
                <li>Take 1ml of each O/N culture and incubate them in corresponding erlenmeyers</p>
                <li>Measure immediately OD600 at t=0h</p>
                <li>Put the erlenmeyers at 37°C 250 rpm</p>
                <li>Carry on until you reach OD600 = 0,5. When it is the case, split the culture in two using a second erlenmeyer for each culture. For each sequence to be characterised, you should have two erlenmeyers with the same volume of culture. Note the time at which the OD600 = 0,5 has been reached out. We decided to do this step because cells grow very slowly at cold temperatures.</br>

                Cultures can grow slower than others so you need to  proceed the next step separately for each culture. For example if the seq 7 (BBa_K2282011) culture takes 4h to reach OD600 = 0,5 and the seq 1 (BBa_K2282005) culture takes 6h to reach OD600 = 0,5, do not wait for the seq 1 (BBa_K2282005) to grow and immediately take care of the seq 7 (BBa_K2282011).</p>

                <li>Put one erlenmeyer at 37°C and the other in the other incubator that you prepared at the wished temperature. make sure the rpm are the same for both.  Wait 18-24h.</p>
              </ol>

            <p class= "txt"><strong>Day 2: first color measurement</strong></p>
              <ol>
                <li>After 18-24h of incubation, take off both erlenmeyers per transformed culture and measure OD600 for both. </p>
                <li>Pellet 1,5ml of each culture and take photos.</p>
                <li>You can proceed protein extraction protocol at this point to measure the quantity of protein expressed as well.</p>
                <li>Put the erlenmeyers in their corresponding incubators and wait 20-24h again.</p>
              </ol>

            <p class= "txt"><strong>Day 3: final color measurement</strong></p>
              <ol>
                <li>Repeat steps done at day 2</p>
              </ol>

              Make sure to replicate your experiment</p>

            <p class= "txt"><strong>Source:</strong>This protocol has been adapted by the Takara’s pCold protein expression inducing, since it uses a similar technology as us to induce cold expression of proteins.</br>
              Link: <a href="http://www.clontech.com/xxclt_ibcGetAttachment.jsp?cItemId=10214&embedded=true">http://www.clontech.com/xxclt_ibcGetAttachment.jsp?cItemId=10214&embedded=true</a>></p>

            <p class= "txt"><strong>Reagents:</strong></p>
              <ul>
                <li>If  protein extraction, see protocol 13</li>
                <li>Bacterial culture of choice</li>
              </ul>

            <p class= "txt"><strong>Materials:</strong></p>
                <ul>
                  <li>Sterile erlenmeyers</li>
                  <li>Incubator Shaker (New Brunswick Scientific Co .INC Series 25)</li>
                  <li>Incubator Shaker for low temperatures (Multitron Strandard Infors HT)</li>
                  <li>Spectrometer (Beckman DU530)</li>
                </ul>
              </p>




              <h2 class="display-3">Protocol 16 : Heat-response characterisation through colorimetry</h2>
              <p class= "txt">This protocol is the same as the protocol 14, but as we are going to use higher temperatures, little modifications have been done. For the mRFP, instead of extraction, you could do fluorescence measurement in microplates  :</p>
              <p class= "txt"><strong>Day 0 : Pre-Incubation</strong></p>
              As for the competent cell protocol, an OD600 kinetic will be made and you will need the bacteria to reach OD600 = 0,5. To this end making a pre-culture is important to reduce the time you wait for the culture to reach this OD600 (warm-up). We had many troubles reaching the right OD600 without the warm-up so we advise you to do this step.
              <ol>
                <li>Incubate 100µl of transformed bacteria in glycerol stock in 15ml sterile falcons containing 3ml of liquid LB with corresponding antibiotics. </p>
                <li>Put the falcons at 37°C 250 rpm O/N</p>
              </ol>

              <p class= "txt"><strong>Day 1 : Launch of characterisation</strong></p>
              <ol>
                <li>Put one incubator at desired cold temperature to make it ready for the incubation step.</p>
                <li>At this point you can either, like the cold-response characterisation, make OD600 measurement until it reaches 0,5 or directly incubate 1ml of your O/N culture into corresponding erlenmeyers (50ml). You should have two erlenmeyers per culture you wish to characterise that have the same volume of LB medium.</p>
                <li>Put one erlenmeyer at 37°C and the other in the other incubator that you prepared at the wished temperature. make sure the rpm are the same for both.  Wait 18-24h.</p>
              </ol>

              <p class= "txt"><strong>Day 2: first color measurement </strong></p>
              <ol>
                <li>After 18-24h of incubation, take off both erlenmeyers per transformed culture and measure OD600 for both.</p>
                <li>Pellet 1,5ml of each culture and take photos.</p>
                <li>You can proceed protein extraction protocol at this point to measure the quantity of protein expressed as well.</p>
                <li>Put the erlenmeyers in their corresponding incubators and wait 20-24h again.</p>
              </ol>

              <p class= "txt"><strong>Day 3: final color measurement</strong></p>
              <ol>
                <li>Repeat steps done at day 2.
              </ol>
              Make sure to replicate your experiment.</p>
              <strong>Source:</strong> we decided to make a similar protocol as the one for the cold-response characterisation, to uniformise the process. </p>
              <strong>Reagents:</strong>
              <ul>
                <li>If  protein extraction, see protocol 13</li>
                <li>Bacterial culture of choice</li>
              </ul>

              <strong>Materials:</strong>
              <ul>
                <li>Sterile erlenmeyers</li>
                <li>Incubator Shaker (New Brunswick Scientific Co .INC Series 25)</li>
                <li>Incubator Shaker for low temperatures (Multitron Strandard Infors HT)</li>
                <li>Spectrometer (Beckman DU 530)</li>
              </ul>




            <h2 class="display-3">Protocol 17: Gel extraction</h2>
            <p class= "txt"><strong>Aim:</strong>purify DNA directly from the gel after migration</p>
            The Qiaquick kit from Qiagen was used to carry out gel extraction. We didn’t obtained great results by using this kit so we recommend to avoid gel extraction as much as possible.</p>
            <ol>
              <li>Cut the bands with the scalpel and put them into separate eppendorf tubes.</p>
              <li>Weight the gel slices.</p>
              <li>Add 3 volumes of Buffer QG for 1 volume of gel (100mg of gel = 100µL).</p>
              <li>Heat into the heat bath at 50°C for 10 mins. Vortex often. The gel must be entirely dissolved.</p>
              <li>Add 1 gel volume isopropanol.</p>
              <li>Put this mix into the spin column (700µL by 700µL) and centrifuge 1min at 13000 rpm. Throw the flow-through. Do it again until everything has been centrifuged.</p>
              <li>Wash with 700µL Buffer PE and centrifuge for 1min at 13000rpm. Discard flow-through.</p>
              <li>Place the column into a clean eppendorf tube.</p>
              <li>To elute DNA, add 50µL of nuclease free water (or EB buffer) to the center of the matrix and centrifuge for 1 min at 13000rpm.</p>
            </ol>

            <strong>Source:</strong> <a href="https://www.qiagen.com/fr/shop/sample-technologies/dna/dna-clean-up/qiaquick-gel-extraction-kit/#orderinginformation">https://www.qiagen.com/fr/shop/sample-technologies/dna/dna-clean-up/qiaquick-gel-extraction-kit/#orderinginformation</a></p>
            <strong>Reagents :</strong>
            <ul>
              <li>QIAquick Gel Extraction Kit (Qiagen 28704)</li>
              <li>Agarose gel with DNA bands</li>
              <li>Nuclease free water</li>
            </ul>
            <strong>Materials:</strong>
            <ul>
              <li>Analytical balance (Analytical balance )</li>
              <li>Sterile scalpel </li>
              <li>Centrifuge (Eppendorf 5417)</li>
            </ul>




          <h2 class="display-3">Protocol 18 : Kinetic in 96 well plates and Tecan infinite 200 pro - iGEM Bettencourt collab</h2>
          <p class= "txt"><strong>Day 0 : Preparation of petri dishes</strong></p>
          <ol>
            <li>Make sure to inoculate any transformed culture you want to test out on appropriate selection plates before the experiment.</li></br>
            <li>Incubate O/N at 37°C</li></p>
          </ol>

          <p class= "txt"><strong>Day 1 : Wells preparation and measurement </strong></p>
          Make sure to make triplicates and work in sterile conditions.</p>
          <ol>
            <li>For each well you want to use, prepare 300µl of liquid medium with corresponding antibiotics. Use Multi-channel pipet to put the medium in the plates quickly.</li>
            <li>Once the medium is put in the wells, scrap a colony from the selection plates you want to caracterise with a pipet tip and shake gently in the corresponding well.</li></p>
            <li>Cover the plate and put it in the microplate reader with the parameters of your choice (shaking, temperature, cycle duration..)</li></p>
            <li>Launch the device and analyse the results once it is done.</li></p>
          </ol>
          <p class= "txt"><strong>Source:</strong>This protocol has been realised with the collaboration of the iGEM Bettencourt team, who provided us very useful Tecan microplate readers to add data to the colorimetry we made earlier.</p>
          <p class= "txt"><strong>Reagents:</strong>
            <ul>
              <li>LB medium with corresponding antibiotics (see protocol 1)</li>
              <li>Colonies of transformed bacteria on petri dishes</li>
              <li>96 well plates</li>
            </ul>

          <p class= "txt"><strong>Materials:</strong>
            <ol>
              <li>Microplate reader (Tecan Infinite 200 pro)</li></br>
              <li>Multichannel pipet</li></br>
              <li>Sterile tips</li></p>
            </ol>



          <h2 class="display-3">Protocol 19: DNA fragments resuspension</h2>
          <p class= "txt"><strong>Aim:</strong> resuspending your dried DNA gene fragments</p>

          <ol>
            <li>Centrifuge the tube for 3-5 sec at a minimum of 3000g to ensure the material is in the bottom of the tube</li></p>
            <li>Add TE to reach a final concentration of 10ng/µl.</li></p>
            <li>Vortex briefly.</li></p>
            <li>Incubate at 50°C for 20 min.</li></p>
            <li>Briefly vortex and centrifuge.</li></p>
          </ol>

          <strong>Source:</strong> <a href="https://eu.idtdna.com/pages/docs/default-source/user-guides-and-protocols/gblocks-amplification.pdf?sfvrsn=12">https://eu.idtdna.com/pages/docs/default-source/user-guides-and-protocols/gblocks-amplification.pdf?sfvrsn=12</a></p>
          <strong>Reagents:</strong></p>
          <ul>
            <li>TE buffer (provided by our school Sup’Biotech)</li>
          </ul>
          <strong>Materials:</strong></p>
          <ul>
            <li>Centrifuge (Eppendorf 5417)</li>
            <li>Waterbath (Julabo 5M)</li>
            <li>Vortex (vortex-genie winn)</li></p>
          </ul>

  </body>
</html>
